src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -58,26 +82,24 @@ This configuration option defines how your Astro project is deployed to Cloudfla
 
```

Comment:
I'm still not the biggest fan of how this looks in the final docs, however I do lack a better idea.  Maybe this has to be a custom component, which renders similar to JSDoc Hover Overlay in VSCode.

This applies to all occurences.

Reply:
This is very similar to our Configuration Reference docs in appearance. Do you mean that you don't like that either, or that there's a difference between this and that you'd like to see applied here?

If it's just "this isn't a great visual" in general, then yes we can play with some styling. I will say that accessibility is a top concern for us, as well as navigability. Right now this checks those boxes (all options appear in the right sidebar TOC, text is certainly accessible etc.)

Reply:
I just double checked and it definitely looks better in the Reference docs _(see screenshots)_. I'm not 100% happy with that either, but it is definitely a lot better, to what we see in the Cloudflare adapter right now. So I would be fine if we change the Cloudflare page to match the reference docs design. If we really need to do more than that, I'm unsure yet.

(Cloudflare example of `mode` setting)
![Capture-2023-12-28-215921](https://github.com/withastro/docs/assets/45965090/0647d26f-fb86-460b-8f3f-72d97a6201c0)
(Cloudflare example of `runtime` setting)
![Capture-2023-12-28-220050](https://github.com/withastro/docs/assets/45965090/298e8680-962b-4d02-80dd-1359f17bcec1)
(Reference example of `redirects` setting)
![Capture-2023-12-28-220017](https://github.com/withastro/docs/assets/45965090/875424d0-686b-4fd7-8c91-0bfddbdd2472)

Reply:
The good news is it's much easier to do here where we can edit live in place and see what the changes look like! I do remember early on in the Cloudflare guide I had to switch up headings etc. on you because I knew what that was going to actually look like live. 

My recommendation for *this* PR is we don't let this stop us here (since it's literally what we had before), but I'm happy after all these code formatting etc. changes are made for us to do a deep Cloudflare PR and get it looking the way you want, then we can use that to remodel the other adapters.